web_widget_one2many_product_picker/README.rst

```diff
@@ -177,6 +177,13 @@ Other example for 'purchase.order.line' fields:
         </form>
     </field>
 
+
+Boostrap Modifications:
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The product picker view container have a custom media queries space adding a new screen size called 'xxl' (>= 1440px) and modifies the columns to have 24 instead of 12.
```

Comment:
Is this text still needed then?

Reply:
The text was updated in the last "force-pushed"